src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -853,6 +875,14 @@ def downgrade():
     bind = op.get_bind()
 
     downward_migrate_notices(bind)
+
+    bind.execute(
+        text(
+            """
+            DELETE FROM experiencetranslation;
+            """
+        )
+    )
```

Comment:
We won't be able to delete experience configs in the downgrade if we don't delete their experience translations first -